src/test/kotlin/no/nav/syfo/dialogmotestatusendring/DialogmoteStatusendringServiceSpek.kt

```diff
@@ -0,0 +1,88 @@
+package no.nav.syfo.dialogmotestatusendring
+
+import io.ktor.util.*
+import no.nav.syfo.dialogmotestatusendring.domain.DialogmoteStatusendring
+import no.nav.syfo.dialogmotestatusendring.domain.DialogmoteStatusendringType
+import no.nav.syfo.dialogmotesvar.domain.DialogmoteSvartype
+import no.nav.syfo.dialogmotesvar.domain.KDialogmotesvar
+import no.nav.syfo.personoppgave.domain.PersonOppgaveType
+import no.nav.syfo.testutil.*
+import org.amshove.kluent.shouldBeEqualTo
+import org.amshove.kluent.shouldNotBe
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.time.OffsetDateTime
+import java.util.*
+
+@InternalAPI
+object DialogmoteStatusendringServiceSpek : Spek({
+    describe("Finish personoppgave when receiving an endring in dialogmotestatus") {
+        val externalMockEnvironment = ExternalMockEnvironment()
+        val database = externalMockEnvironment.database
+        val dialogmoteUuid = UUID.randomUUID()
+        val kDialogmotesvar = KDialogmotesvar(
+            personident = UserConstants.ARBEIDSTAKER_FNR.value,
+            type = DialogmoteSvartype.KOMMER_IKKE.name,
+            dialogmoteUuid = dialogmoteUuid.toString(),
+            virksomhetsnummer = UserConstants.VIRKSOMHETSNUMMER.value,
+        )
+
+        afterEachTest {
+            database.connection.dropData()
+        }
+
+        it("Finish personoppgave when dialogmote has a referat") {
+            database.connection.createPersonOppgave(
+                kDialogmotesvar = kDialogmotesvar,
+                type = PersonOppgaveType.DIALOGMOTESVAR,
+            )
+            val dialogmoteStatusendring = DialogmoteStatusendring(
+                personIdent = UserConstants.ARBEIDSTAKER_FNR,
+                type = DialogmoteStatusendringType.FERDIGSTILT,
+                endringTidspunkt = OffsetDateTime.now(),
+                dialogmoteUuid = dialogmoteUuid,
+                veilederIdent = UserConstants.VEILEDER_IDENT,
+            )
+
+            val connection = database.connection
+            processDialogmoteStatusendring(connection, dialogmoteStatusendring)
```

Comment:
Burde det ikke være en mock-kafka-consumer her?

Reply:
Er ikke noe kafka i akkurat denne funksjonen/fila, men du har rett i at vi ikke har tester for det. Tenkte å ta det i neste runde.